### ISSUE TITLE
Test with Mockery\Expectation will be marked as risky.

### ISSUE THREAD

**Issue by zingimmick** [pestphp/pest#23]
Pest will mark the following test as risky. PHPUnit will mark it as passed.

```php
use Illuminate\Log\Logger;
use Mockery as m;
use Monolog\Logger as Monolog;

test('example', function (): void {
    $writer = new Logger($monolog = m::mock(Monolog::class));
    $monolog->shouldReceive('error')->once()->with('foo', []);

    $writer->error('foo');
});
```

**Comment by nunomaduro** [comment]
On it...

**Comment by nunomaduro** [comment]
https://github.com/pestphp/pest/commit/415f5719101b30c11d87f74810a71686ef2786c6

**Comment by nunomaduro** [comment]
Fixed on `v0.1.5`.
